app/components/shared/document_status_tag_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render "govuk_publishing_components/components/tag", {
+  text: state,
```

Comment:
Just reading this I thought you were writing the `edition.state` onto the page literally, but I see below that this is actually a lookup in the i18n file. I think it might be beneficial rename the state function below to something else so as not to overload the term.